Cubical/Algebra/CommMonoid/CommMonoidProd.agda

```diff
@@ -0,0 +1,38 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommMonoid.CommMonoidProd where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Algebra.Monoid
+open import Cubical.Algebra.Semigroup
+open import Cubical.Algebra.CommMonoid.Base
+
+
+open CommMonoidStr
+open IsCommMonoid hiding (rid ; lid)
+open IsMonoid hiding (rid ; lid)
+open IsSemigroup
+
+
+private
+  variable
+    𝓁 𝓁' : Level
+
+
+CommMonoidProd : ∀ {ℓ ℓ'} → CommMonoid ℓ → CommMonoid ℓ' → CommMonoid (ℓ-max ℓ ℓ')
```

Comment:
The variables for the universe level are symbols different from '\ell'. If they were '\ell', the implicit arguments to 'CommMonoidProd' could be dropped.